ci/build_setup.sh

```diff
@@ -65,6 +65,7 @@ export BAZEL_BUILD_OPTIONS="--strategy=Genrule=standalone --spawn_strategy=stand
   --verbose_failures ${BAZEL_OPTIONS} --action_env=HOME --action_env=PYTHONUSERBASE \
   --jobs=${NUM_CPUS} --show_task_finish ${BAZEL_BUILD_EXTRA_OPTIONS}"
 export BAZEL_TEST_OPTIONS="${BAZEL_BUILD_OPTIONS} --test_env=HOME --test_env=PYTHONUSERBASE \
+  --test_env=UBSAN_OPTIONS=print_stacktrace=1 \
```

Comment:
This could optionally be made local to the ASAN/UBSAN invocation, but looking at docs (https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html) it seems harmless to do it here as well.